Trippy/Utils/MapUtils.m

```diff
@@ -29,4 +31,18 @@ + (UIImage *)getStaticMapImage:(CLLocationCoordinate2D)location width:(int)width
     return image;
 }
 
++ (NSString *) generateDirectionsApiUrl:(LocationCollection *)collection
+                                          origin:(Location *)origin
+                                        optimize:(BOOL)optimize
+                                   departureTime:(NSDate *)departureTime {
+    // TODO: Enable via waypoints instead of just stopovers
+    NSString *stops = optimize ? @"optimize:true" : @"";
```

Comment:
Curious what "optimize" means here. Is this just a testing thing on your part, or do you get a significant change in results if you use this parameter? If so, you might want to rename your input param to something more reflective (for example, "optimizeWaypointPath" or some such).

Reply:
It's pretty significant, it's what Google Directions uses to decide if it's TSP or if the waypoints should be considered in set order. Will update input name though!